test/testcafe/spec/ChallengeFactorEmail_OTP_spec.js

```diff
@@ -33,8 +33,8 @@ test
     const challengeFactorPageObject = await setup(t);
     const pageTitle = challengeFactorPageObject.getPageTitle();
     await t.expect(pageTitle).contains('Email link');
-    const pageSubTitle = challengeFactorPageObject.getPageSubTitle('.okta-form-subtitle');
-    await t.expect(pageSubTitle).contains('An email has been sent to you. Please click the link in your email or enter the code from that email below.');
+    await t.expect(challengeFactorPageObject.getTextContent('.email-verification-description'))
```

Comment:
I am curious why we are adding a class specific to this view `email-verification-description` for the subtitle here ? Is it to avoid the styles applied to `okta-form-subtitle`?

Reply:
This particular view is no more the subtitle. For majority of the views, UX wants  subtitle to be styled left aligned and here it conflicts.